src/interfaces/ffi/structs.rs

```diff
@@ -0,0 +1,102 @@
+// ---------------------------------------
```

Comment:
this file does not seem to be used anymore

Reply:
Yes. Right. That was my attempt at using opaque objects through the FFI but it fails. Removing it